src/installer/corehost/common/pal.windows.cpp

```diff
@@ -96,11 +96,11 @@ void* pal::map_file_readonly(const pal::string_t& path, size_t &length)
     length = (size_t)fileSize.QuadPart;
 
     HANDLE map = CreateFileMappingW(file, NULL, PAGE_READONLY, 0, 0, NULL);
+    CloseHandle(file);
 
     if (map == NULL)
     {
         trace::warning(_X("Failed to map file. CreateFileMappingW(%s) failed with error %d"), path.c_str(), GetLastError());
```

Comment:
`CloseHandle` above may alter `GetLastError`. If that happens, the last error in the log will be bogus.